README.md

```diff
@@ -2,139 +2,117 @@
-  - [AWS Device Farm](#aws-device-farm)
+  - [Main usage](#main-usage)
+  - [Advanced usage](#advanced-usage)
+  - [Using other e2e frameworks](#using-other-e2e-frameworks)
+  - [Comparing](#comparing)
+  - [Exploiting measures](#exploiting-measures)
+  - [Running in CI](#running-in-ci)
+    - [AWS Device Farm](#aws-device-farm)
 - [Flipper Plugin](#flipper-plugin)
-  - [Install](#install)
 - [CLI](#cli)
-- [Getting FPS Data](#getting-fps-data)
+  - [Via Custom script](#via-custom-script)
 - [Contributing](#contributing)
   - [web-reporter](#web-reporter)
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
 ## Getting started with the automated profiler
 
-1. Install the profiler
+### Main usage
 
-```sh
-yarn add --dev @perf-profiler/e2e
-```
+TODO: providing a sample app here would be nice (either a production one like Twitter, or a sample APK)
 
-2. Create a TS script including a performance test
+1. Install the profiler `yarn add --dev @perf-profiler/e2e`
 
-For instance, here we'll be testing the start up performance of the app for 10 iterations:
+2. Create a TS script including a performance test, for instance `start.ts`
 
 ```ts
-// performance.ts
-import { execSync } from "child_process";
+import { AppiumDriver } from "@bam.tech/appium-helper";
 import { TestCase, measurePerformance } from "@perf-profiler/e2e";
 
-// `npx @perf-profiler/profiler getCurrentApp` will display info for the current app
-const bundleId = "com.reactnativefeed";
-
-const stopApp = () => execSync(`adb shell am force-stop ${bundleId}`);
-const startApp = () =>
-  execSync(
-    `adb shell monkey -p ${bundleId} -c android.intent.category.LAUNCHER 1`
-  );
-
-const startTestCase: TestCase = {
-  duration: 10000,
-  beforeTest: () => {
-    stopApp();
-  },
-  run: () => {
-    startApp();
-  },
-};
+const runTest = async () => {
+  const driver = await AppiumDriver.create({
+    // `npx @perf-profiler/profiler getCurrentApp` will display info for the current app
+    appPackage: "com.example",
+    appActivity: "com.example.MainActivity",
+  });
 
-const test = async () => {
-  const { measures, writeResults } = await measurePerformance(
-    bundleId,
-    startTestCase,
-    10
-  );
+  const testCase: TestCase = {
+    beforeTest: async () => {
+      driver.stopApp();
+    },
+    run: async () => {
+      // run is where measuring will happen
+      driver.startApp();
+      await driver.findElementByText("KILL JS");
+    },
+    // Duration is optional, but helps in getting consistent measures.
+    // Measures will be taken for this duration, regardless of test duration
+    duration: 10000,
+  };
+
+  const { writeResults } = await measurePerformance(bundleId, testCase);
   writeResults();
 };
 
-test();
-```
-
-3. Run your script:
-
-```sh
-npx ts-node performance.ts
+runTest();
 ```
 
-4. Open the JSON file generated in the web profiler:
+3. Run `npx appium` in one tab
+4. Run `npx ts-node start.ts` in a separate tab
```

Comment:
Should we explain that we can use jest if it is already installed on the project?